pages/whats-new/product-roadmap.md

```diff
@@ -7,35 +7,66 @@ layout: styleguide
+<!-- TODO: Make these into issues; add roadmap project board
+You can also <a href="https://github.com/uswds/uswds/milestone/52" class="">view our product roadmap goals on GitHub</a>. -->
+
+<h2>2020 Roadmap</h2>
+
+<div class="maxw-tablet">
+  <ul class="site-roadmap-list">
+    <li class="site-roadmap-list__item"><span class="flex-fill">Build most-requested new components</span> <span class="usa-tag label-in-progress flex-auto">In progress</span>
+      <ul class="site-roadmap-list__sublist">
+        <li class="site-roadmap-list__item"><span class="flex-fill">Card</span> <span class="usa-tag label-in-progress">In progress</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Breadcrumb</span> <span class="usa-tag label-next">On deck</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Character limit</span> <span class="usa-tag label-in-progress">In progress</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Autocomplete combobox</span> <span class="usa-tag label-in-progress">In progress</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Button group</span> <span class="usa-tag label-in-progress">In progress</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Date picker</span>  <span class="usa-tag label-next">On deck</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Date range picker</span>  <span class="usa-tag label-next">On deck</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Time picker</span>  <span class="usa-tag label-next">On deck</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Date and time picker</span> <span class="usa-tag label-next">On deck</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Tooltip</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Document upload</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Progress bar</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Anchor</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Tabs</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Icons</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Button icon</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
+        <li class="site-roadmap-list__item"><span class="flex-fill">Paginator</span> <span class="usa-tag label-upcoming">Upcoming</span></li>
```

Comment:
I think this is more commonly referred to as "Pagination."

Reply:
I'll update